docs/src/architecture.md

```diff
@@ -283,12 +417,30 @@ Payload example for POST:
 }
 ```
 
+### Permissions
+The *License Manager API* needs an authentication provider (OIDC) to manage the permissions for the users.
+
+We recommend using the *Keycloak* as the OIDC provider, but any provider that supports the *OIDC* protocol can be used.
+
+Each endpoint is protected using [Armasec](https://github.com/omnivector-solutions/armasec), which manages the permissions
+needed to access each endpoint. The permissions must be available as a claim in the JWT token.
```

Comment:
Perhaps mention that there are guides for getting started with Keycloak (or Auth0) in the Armasec docs.

Reply:
Updated this section.